doc/src/devdocs/ssair.md

```diff
@@ -1,5 +1,54 @@
 # Julia SSA-form IR
 
+Julia uses a static single assignment intermediate representation ([SSA IR](https://en.wikipedia.org/wiki/Static_single-assignment_form)) to perform optimization.
+This IR is different from LLVM IR, and unique to Julia.
+It allows for Julia specific optimizations.
+The SSA part means that in the IR, every variable is assigned to exactly once and this definition occurs before all uses of the variable. The transformation to IR (SSA conversion only happens after inference) is performed by code lowering, and in this representation there are a few changes worth highlighting.
```

Comment:
```suggestion
```
This sentence is mostly false. SSA is mostly a claim about reachability and side-effects of our instructions, and that every instruction has a unique numbering that identifies its side effects and result. Those facts are true and present prior to inference. No new SSA properties are added after inference.

Reply:
Thanks, it has grammatical errors as well.